src/BUILD

```
@@ -102,6 +81,14 @@ filegroup(
     visibility = ["//visibility:public"],
 )
 
+filegroup(
+    name = "docs_assets",
+    srcs = glob([
+        "assets/css/*.css",
```

Something more generic? Whatever the syntax is.
```suggestion
        "assets/**/*",
```

---

Fixed